src/components/inputs/radio-input/radio-option.js

```diff
@@ -66,6 +67,16 @@ Option.propTypes = {
     PropTypes.element,
     PropTypes.func,
   ]).isRequired,
+  components: PropTypes.shape({
+    WrapperComponent: (props, propName) => {
```

Comment:
is this really necessary? Can't we just make it optional?

Reply:
You mean the `children` prop right?